### ISSUE TITLE
0.12.5: BUILD_STATICALLY_LINKED_EXE=OFF and static library is build and installed

### ISSUE THREAD

**Issue by kloczek** [editorconfig/editorconfig-core-c#77]
Looksl like despite passing to cmake `BUILD_STATICALLY_LINKED_EXE=OFF` libeditorconfig_static.a is build and installed.

**Comment by xuhdev** [comment]
Could you point out the problem? Is `libeditorconfig.so` also there?

**Comment by kloczek** [comment]
Yes it is however libeditorconfig_static.a is build and install as well despite what is passed in `BUILD_STATICALLY_LINKED_EXE`

**Comment by xuhdev** [comment]
Is the executable dynamically linked? This is what the option `BUILD_STATICALLY_LINKED_EXE` means. It doesn't change what happens to the library build.
